ChangeLog

```diff
@@ -27,6 +28,7 @@
  * Removed deprecated SobolIndicesAlgorithm::Generate method
  * Removed deprecated NumericalScalar, NumericalComplex types
  * Removed deprecated Function constructors
+ * Deprecated FunctionalChaosRandomVector in favor of FunctionalChaosSensitivity
```

Comment:
@jschueller This class should not be removed, as it is not only a (bad) interface to sensitivity analysis based on chaos. It also provides a sampler based on the orthogonal measure, ie of Y=g(Z) instead of Y=f(T(Z)) where T is the transformation mapping Z into X, where Z is distributed according to the orthogonality measure. It makes a huge difference eg when T relies on RosenblattEvaluation.